ansible/inventory

```diff
@@ -1 +1 @@
-192.168.20.183 ansible_user=pi
+192.168.1.244 ansible_user=pi
```

Comment:
People are going to change this based on their system - no single IP address will be correct - perhaps we should just leave it as an empty file (with a comment showing format). Thoughts?